app/tests/test_world_map_cache.py

```
@@ -0,0 +1,275 @@
+                    return country_df
+                return []
+
+            mock_supabase.get_data.side_effect = _get_data_side_effect
+
+            resp1 = self.client.get(f"/ja/{self.year}/world_map")
+            resp2 = self.client.get(f"/ja/{self.year}/world_map")
+
+            self.assertEqual(resp1.status_code, 200)
+            self.assertEqual(resp2.status_code, 200)
+
+            # 1回目: 生成（save が呼ばれる）
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)